python/tests/unit/arcticdb/version_store/test_update.py

```diff
@@ -127,10 +127,13 @@ def test_update_repeatedly_dynamic_schema_hashed(
 
     idx = pd.date_range("1970-01-01", periods=100, freq="D")
     l = len(idx)
-    df = pd.DataFrame(
-        {"a": range(l), "b": range(1, l + 1), "c": range(2, l + 2), "d": range(3, l + 3), "e": range(4, l + 4)},
-        index=idx,
-    )
+    df = pd.DataFrame({
+        "a": np.arange(l, dtype='float'), 
+        "b": np.arange(1, l + 1, dtype='float'),
+        "c": np.arange(2, l + 2, dtype='float'),
+        "d": np.arange(3, l + 3, dtype='float'),
+        "e": np.arange(4, l + 4, dtype='float')
```

Comment:
Also curious why `range()` stopped working.

Also, we don't have `black` set up yet, but please use double quotes.

Reply:
`df.update` used to convert the the `int` columns to `float` more often than it should, that got fixed.  Switching to float here so that it works across pandas versions.

Reply:
I'll do the black changes